CODING_STANDARDS.md

```diff
@@ -15,7 +15,17 @@ We adhere to the `pep8` standard, using `flake8`, `pylint` and `mypy` to catch f
 
 ## Solidity
 
-As of now, the solidity code does not follow consistent style conventions. Nevertheless, new solidity code must adhere to the [solidity coding standards](https://docs.soliditylang.org/en/develop/style-guide.html). Old code will progressively be updated to converge towards the recommended style.
+Solidity code must adhere to the [solidity coding standards](https://docs.soliditylang.org/en/develop/style-guide.html), with the following amendments:
+- Test functions must end with the suffix `Test` (e.g. `reverseBytesTest`)
+- Private/internal state variables and functions must have an underscore prefix (e.g. `_myInternalFunction`)
+- Function parameters must not be prefixed with an underscore
+- Interface names must have a capital I prefix (e.g. `IERC20`)
+- Library names must have a capital L prefix (e.g. `LPairing`)
+- Test contract names must have a `Test` prefix (e.g. `TestMyContract`)
+- Event names must be prefixed by `Log` (e.g. `LogDeposit`)
+- Contract names may not be PascalCase if using PascalCase is introducing confusions in the name (e.g. `BLS12377.sol` vs `BLS12_377.sol`). PascalCase should be used whenever possible
+
+**Note:** Some of the files of the current code base may not fully comply with the coding standards above. Old code will progressively be updated to converge towards the recommended style.
```

Comment:
More precisely, the `snake_case` convention remains broadly used in function bodies (we still have mixed styles there). The function signatures and contract's state variables/constants however should be all fine now